pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -397,6 +397,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			Rules: []authorizationapi.PolicyRule{
 				// push and pull images
 				authorizationapi.NewRule("get", "update").Groups(imageGroup).Resources("imagestreams/layers").RuleOrDie(),
+				// allow auto-provisioning when pushing an image that doesn't have an imagestream yet
```

Comment:
> do we always want builders to have this power?

I can't think of a reasonable reason not to.  If a cluster-admin comes up with one, he can change the default role.